docs/notebooks/proton_radiography_particle_tracing.ipynb

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
<p>This sounds confusing, I guess it'll make more sense once I've gone through the code...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
It will - but that sort of defeats the point of an example I guess! I've added a brief summary of the steps taken by the algorithm to the notebook.